conf/messages

```diff
@@ -44,12 +44,11 @@ error.business-name.max-length                                  = Business name
 error.business-name.invalid-character                           = Business name can only include characters listed.
 
 # Business start date
-business.start-date.title                                       = When did your business start trading?
-business.start-date.heading                                     = When did your business start trading?
-business.start-date.caption                                     = Sole trader
-business.start-date.line-1                                      = The date your business started trading can be today, in the past or up to 7 days in the future
-business.start-date.line-2                                      = This is the date we’ll use to calculate Class 2 National Insurance charge, if appropriate.
-business.start-date.hint                                        = For example, 17 4 2018.
+business.start-date.title                                       = Start date for sole trader business
+business.start-date.heading                                     = Start date
+business.start-date.caption                                     = Your sole trader business
+business.start-date.paragraph                                   = We need to know the exact start date.
+business.start-date.hint                                        = For example, 27 9 2024
```

Comment:
```suggestion
business.start-date.hint                                        = For example, 27 9 {0}
```
Potentially look into having the year as a parameter, similar to what was done on the agent side of things.
I believe hint text should be a valid input, but being hard coded, it will not be valid in a few years.

Reply:
Oh very good point, I agree.
I've used the start date limit year because we could be in June 2024, and using current year would be too far in the future